game/scripts/vscripts/components/boss/ngp.lua

```diff
@@ -26,15 +26,14 @@ function NGP:PlayerVote (eventSourceIndex, args)
   local id = eventSourceIndex.id
   local option = eventSourceIndex.option
   local team = ShortTeamName(PlayerResource:GetTeam(playerID))
-  local item = NGP.activeItems[tonumber(id)]
```

Comment:
You don't need to do that, in fact you shouldn't. `item` is a reference to `NGP.activeItems[tonumber(id)]`. `item` and `NGP.activeItems[tonumber(id)]` are effectively the same. But `item` is shorter and easier to read.
I didn't mean you should replace it.